lib/smart_answer_flows/uk-benefits-abroad.rb

```diff
@@ -552,9 +552,9 @@ def define
 
         next_node do |response|
           if calculator.going_abroad && response == "esa_under_a_year_medical"
-            outcome :esa_going_abroad_under_a_year_medical_outcome # A27 going_abroad
+            outcome :esa_going_abroad_under_a_year_medical_outcome
           elsif calculator.going_abroad && response == "esa_under_a_year_other"
-            outcome :esa_going_abroad_under_a_year_other_outcome # A28 going_abroad
+            outcome :esa_going_abroad_under_a_year_other_outcome
```

Comment:
Nitpick: you've removed these comments, but not the one on line 559.
Happy to keep them or remove them either way, but we should probably be consistent throughout the file.